contracts/LimitOrder.sol

```diff
@@ -57,13 +59,19 @@ contract LimitOrder is ILimitOrder, StrategyBase, BaseLibEIP712, SignatureValida
 
     receive() external payable {}
 
+    /// @notice Called by owner only
```

Comment:
nit: `called by XXX only` feels like it is **expected to be called by XXX**, how about
```suggestion
    /// @notice Only owner can call
```

Reply:
I'm ok with both. Adjusted in 4fed1e9c109a6478d2a92f53649d77877b9f9668.